docs/src/pages/basics/writing-stories/index.md

```diff
@@ -93,17 +93,26 @@ It's up to you to find a naming/placing scheme that works for your project/team.
 
 ## Loading stories
 
-Stories are loaded in the `.storybook/config.js` file.
+Stories are loaded in the `.storybook/main.js` file or `.storybook/preview.js` file.
 
 The most convenient way to load stories is by filename. For example, if your stories files are located in the `src/components` directory, you can use the following snippet:
 
+```js
+// .storybook/main.js
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+};
+```
+
+Alternatively you can import all your stories in `preview.js`:
```

Comment:
Do you think the `require.context()` method is still worth mentioning? I would probably just make a straightforward `import` example here:

```js
import 'firstStoryFile';
import 'secondStoryFile';
```

I guess people might still want to do that? Or should we even call that deprecated?

Reply:
Let's bring this up in a Roadmap meeting?

These are excellent ideas @tmeasday